app/models/game.rb

```diff
@@ -22,5 +23,45 @@ def coordinate_conflict?(test_coords)
     false
   end
 
+  def populate_game
```

Comment:
We should probably call this method `populate` since calling `game.populate_game` seems redundant